### PR TITLE
Frontend area layout xml improvements.

### LINKED ISSUE #30549: [Issue] Frontend area layout xml improvements.
This issue is automatically created based on existing pull request: magento/magento2#30539: Frontend area layout xml improvements.

---------

### Description (*)

Remove `class="Magento\Framework\View\Element\Template"` from frontend layout xml files, as `"Magento\Framework\View\Element\Template"` is the default class taken by Magento.

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)

N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by sanganinamrata**
### Description (*)

Remove `class="Magento\Framework\View\Element\Template"` from frontend layout xml files, as `"Magento\Framework\View\Element\Template"` is the default class taken by Magento.

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)

N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#30549: Frontend area layout xml improvements.

### CODE REVIEW

**Comment by rogyar** [comment]
@magento run all tests

**Comment by sidolov** [comment]
@magento create issue

**Comment by rogyar** [comment]
@magento run Integration Tests

**Comment by rogyar** [comment]
@magento run all tests

File: app/code/Magento/Theme/view/frontend/layout/default.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <page layout="3columns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <update handle="default_head_blocks"/>
     <body>
-        <attribute name="id" value="html-body"/>
+      <attribute name="id" value="html-body"/>
```
Comment (fredden) [inline app/code/Magento/Theme/view/frontend/layout/default.xml:11]: This change looks accidental / unintentional.

**Review by fredden — APPROVED** [review]

**Comment by engcom-Delta** [comment]
Hi @sanganinamrata, Thanks for your contribution.Please resolve the conflicts to move ahead on this PR

**Review by engcom-Delta — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Comment by engcom-Dash** [comment]
Hello @sanganinamrata,

Thanks for the contributions!

Please resolve the conflict so we can proceed with this PR.

Thanks.

**Comment by engcom-Dash** [comment]
Hello @sanganinamrata,

This is a reminder to resolve the conflicts.

Thanks!

**Comment by engcom-Dash** [comment]
Closing this PR since it has not been updated in a while. Please feel free to reopen it if needed.

**Pull request closed without merging**